sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -38,14 +38,16 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS_BAK}")
 
   add_library(UnifiedRuntimeLoader ALIAS ur_loader)
+  add_library(UnifiedRuntimeCommon ALIAS ur_common)
```

Comment:
thanks @kswiecicki . How is it built? I built the unified runtime and didnt find it in the resulting binaries or libraries.

Reply:
ur_common is only an `interface` library: https://github.com/oneapi-src/unified-runtime/blob/main/source/common/CMakeLists.txt#L9 This means there is no separate binary, but it will add `ur_common` sources and header to every component that uses it. I believe it might change in the future if it turns out we need to keep some extra shared state.

Reply:
ah I see. Thanks @igchor